comp/logs/agent/config/config_test.go

```diff
@@ -361,7 +369,7 @@ func (suite *ConfigTestSuite) TestMultipleHttpEndpointsInConfig2() {
 		APIKey:           "456",
 		Host:             "additional.endpoint.1",
 		Port:             1234,
-		UseSSL:           true,
+		UseSSL:           &trueVal,
```

Comment:
Absolutely nothing wrong with this, but we also have a [helper for generating pointers](https://github.com/DataDog/datadog-agent/blob/c89808ba9d8c593d5fabe672ee00220418bb5060/pkg/util/pointer/pointer.go#L10) for tests like this:
`pointer.Ptr(true)`

Reply:
Ooo didn't know about that, I'll use that instead